<launch>
    <arg name="goals_file" default="navigation_goals"/>
    <rosparam command="delete" param="/goals" />
    <rosparam command="load" file="$(find challenge_evaluation)/config/$(arg goals_file).yaml" />
    <node name="evaluation" pkg="challenge_evaluation" type="evaluation.py" output="screen" launch-prefix="python3">

        <param name="output_file" value="$(find challenge_evaluation)/config/metrics_reactive.txt" />
        <param name="base_frame" value="base_footprint" />
        <param name="global_frame" value="odom" />
        <param name="goal_tolerance" value="0.15" />
        <param name="min_collision_dist" value="0.12" />
    </node>
</launch>



